java-util/src/main/java/io/druid/java/util/common/DateTimes.java

```diff
@@ -38,6 +39,11 @@ public static DateTime of(String instant)
     return new DateTime(instant, ISOChronology.getInstanceUTC());
   }
 
+  public static DateTime of(String instant, DateTimeFormatter formatter)
```

Comment:
Enforcement to use this method could make it easy to repeatedly create some extra garbage, at times. What about adding a wrapper class like "UtcDateTimeFormatter", which is a very simple wrapper of `DateTimeFormatter` with a single `parse()` method, that just accepts `DateTimeFormatter` and calls `this.formatter = formatter.withChronology(ISOChronology.getInstanceUTC())` in the constructor. And then instead of `DateTimes.of(String, DateTimeFormatter)`, to suggest to use `UtcDateTimeFormatter.parse()`.

Reply:
:+1: